src/code/z_play.c

```diff
@@ -1283,24 +1286,25 @@ void Play_Main(GameState* thisx) {
 
     PLAY_LOG(4556);
 
-    if ((HREG(80) == 10) && (HREG(94) != 10)) {
-        HREG(81) = 1;
-        HREG(82) = 1;
-        HREG(83) = 1;
-        HREG(84) = 3;
-        HREG(85) = 1;
-        HREG(86) = 1;
-        HREG(87) = 1;
-        HREG(88) = 1;
-        HREG(89) = 1;
-        HREG(90) = 15;
-        HREG(91) = 1;
-        HREG(92) = 1;
-        HREG(93) = 1;
-        HREG(94) = 10;
+    if ((R_HREG_MODE == HREG_MODE_PLAY) && (R_PLAY_INIT != HREG_MODE_PLAY)) {
+        R_PLAY_RUN_UPDATE = 1;
```

Comment:
Are these and the ones below booleans? If so `1` -> `true` ?

Reply:
they are, though i guess i left them as 1 cause thats what you would input in the reg editor.
I guess in other cases we opt for code readability over the reg menu perspective. but its interesting to note that some other things that would otherwise be booleans are checked for == 1 (`R_LETTERBOX_ENABLE_LOGS == 1`)

Ill still change it cause I think code readability will be more helpful than the few who choose to actually use this system in the menu